assets/html/index.html

```diff
@@ -26,21 +26,37 @@
 		}
 
 		.green {
-			color: rgb(55 117 59);
+			color: #117733;	/* Tol palette: green */
 		}
 
 		.red {
-			color: rgb(125 42 83); /* dark magenta */
+			color: #882255; 	/* Tol palette: red/magenta */
 		}
 
 		.blue {
-			color: rgb(151 202 234); /* light blue */
+			color: #88CCEE; 	/* Tol palette: cyan */
 		}
 
 		.grey {
 			color: grey;
 		}
 
+		.bg-green {
+			background-color: #117733;		/* Tol palette: green */
+		}
+
+		.bg-red {
+			background-color: #882255; 	/* Tol palette: red/magenta */
+		}
+
+		.bg-blue {
+			background-color: #88CCEE; 	/* Tol palette: cyan */
```

Comment:
```suggestion
			background-color: #88CCEE; 	/* Tol palette: light blue */
```
`cyan` is unclear if it means light blue or the `bluegreen` color